angularSTEP/src/app/user-page/user-page.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="profileLoaded">
+    <div class="profile-header">
+        <img class="profile-pic" src="{{ picUrl }}" alt="Profile pic for {{ displayName }}">
+        <div class="name-header">
+            <h1>{{displayName}}</h1>
+            <h3>@{{username}}</h3>
```

Comment:
I think I mentioned this in a different PR, but if not then please be consistent on spaces before and after double braces.